ocs_ci/helpers/helpers.py

```diff
@@ -4656,6 +4656,35 @@ def get_ceph_log_level(service, subsystem):
     return int(log_value)
 
 
+def is_rbd_default_storage_class(custom_sc=None):
+    """
+    Check if RDB is a default storageclass for the cluster
+
+    Args:
+        custom_sc: custom storageclass name.
+
+    Returns:
+        True : If RBD is set as Default storage class for the cluster else False.
+    """
+    default_rbd_sc = (
+        constants.DEFAULT_STORAGECLASS_RBD if custom_sc is None else custom_sc
+    )
+    kubeconfig = os.getenv("KUBECONFIG")
```

Comment:
No need to use kubeconfig. I meant when you use functions outside tests and collectonly will run into the function and invoke it we need to make sure kubeconfig exist. otherwise we don't need it. Delete this like and the kubeconfig command line switch.

Reply:
Removed KUBECONFIG from run_cmd